### ISSUE TITLE
Don't just say "failed"

### ISSUE THREAD

**Issue by jidanni** [commons-app/apps-android-commons#5086]
Give the user a clue what the problem was. For instance 
- it's the Wikimedia  company's fault.
- it's the user's own fault for uploading such unworthy content.
- it's some unknown fault that we are very very sorry but we admit that we don't know what the problem is.

That way the user would know, if he should
- try again
- wait till tomorrow and try again
- go to a community college class about making better content, and then try again.

**Comment by jidanni** [comment]
![Screenshot_20221022-103017.jpg](https://user-images.githubusercontent.com/1230959/197347903-4e30d0f5-096a-427b-99ba-c9efc3237f25.jpg)
![Screenshot_20221022-103041.jpg](https://user-images.githubusercontent.com/1230959/197347908-1e7bbfd5-b566-4b80-b5cc-b867e145d954.jpg)

**Comment by jidanni** [comment]
P.S., I just sent the Debug .zip file. Check your email.

**Comment by jidanni** [comment]
Upload fine via the website.
https://commons.wikimedia.org/wiki/File:Mysterious_lights_over_North_Pacific_Ocean.jpg

**Comment by nicolas-raoul** [comment]
Other upload platforms (for instance Facebook) also just say "Failed".
But I agree that a "details" link with the complete exception message (and maybe even stacktrace if in a scrollable area) would be helpful. It must be copyable.

**Comment by jidanni** [comment]
Yes, all we ever know is Failed. We can hit the retry button a hundred times but that's all we will ever know. Maybe it should just say "Failed. Please use the upload form on the website instead." With a hyperlink offered too.

**Comment by jidanni** [comment]
In Download/logs we just see e.g.,
>2023-01-01 18:27:26,781 [ERROR] [file-logging-thread-1] [UploadWorker] : java.lang.NullPointerException: The supplied value is null 	at io.reactivex.internal.operators.observable.ObservableOnErrorReturn$OnErrorReturnObserver.onError(ObservableOnErrorReturn.java:80)
